docs/content/extensions.md

```diff
@@ -244,7 +246,8 @@ func (p *PrintlnLogger) Log(ctx context.Context, event logs.EventV1) error {
 	if p.config.Stderr {
 		w = os.Stderr
 	}
-	_, err := fmt.Fprintln(w, event)
+	e, _ := json.Marshal(event)
```

Comment:
nit: rename `e` to `bs` (common convention for referring to a slice of bytes...)

Reply:
nit: we should probably do something with `err` here rather than silently discarding it... not that Marshal should ever fail on an event struct but I don't want someone to complain about this in the future.

Reply:
Sounds good I'll make those changes thank you for the feedback.